schemas/number.json

```diff
@@ -40,7 +45,11 @@
             }
           },
           "additionalProperties": false,
-          "required": ["uncertainValue", "uncertainty", "confidenceInterval"]
+          "required": [
+            "uncertainValue",
+            "uncertainty",
+            "confidenceInterval"
+          ]
```

Comment:
Before we merge, it is important to use `make format` to avoid such changes in whitespaces.